emulators/tnylpo/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           gitlab 1.0
+
+name                tnylpo
```

Comment:
this isn't needed as it will be set automatically from the `gitlab.setup` line (or at least should...)

Reply:
Done.